model/src/main/kotlin/config/RuleViolationResolutionReason.kt

```diff
@@ -21,7 +21,7 @@ package com.here.ort.model.config
 
 enum class RuleViolationResolutionReason {
     /**
-     * The evaluator rule violation leading has been approved.
+     * The patent grant implied by a rule violation is accepted or say an exception is made.
```

Comment:
The "or say" part sounds a bit odd to me. How about: "The implied patent grant is acceptable in this case."